src/rt64lib/shaders/Lights.hlsli

```diff
@@ -64,10 +100,11 @@ float CalculateLightIntensitySimple(uint l, float3 position, float3 normal, floa
 	return sampleIntensityFactor * dot(SceneLights[l].diffuseColor, float3(1.0f, 1.0f, 1.0f));
 }
 
-float3 ComputeLight(uint2 launchIndex, uint lightIndex, float3 rayDirection, uint instanceId, float3 position, float3 normal, float3 specular, const bool checkShadows) {
+float2x3 ComputeLight(uint2 launchIndex, uint lightIndex, float3 rayDirection, uint instanceId, float3 position, float3 normal, float3 specular, float roughness, float3 viewPosition, const bool checkShadows) {
```

Comment:
I find the readability of the return value of this function a bit odd. Could we make this a simple struct instead with two float3s?

Something along the lines of:
```
struct LightResult {
  float3 diffuseColor;
  float3 specularColor;
};
```

That should help make the code that uses all light values easier to read.